src/sarif-converter-21.ts

```diff
@@ -294,24 +286,27 @@ export class SarifConverter21 {
     private convertRuleResultsWithoutNodes(
         resultArray: Sarif.Result[],
         ruleResults: DecoratedAxeResult[],
-        level: CustomSarif.Result.level,
-        properties: DictionaryStringTo<string>,
+        ruleIdsToRuleIndices: DictionaryStringTo<number>,
+        kind: CustomSarif.Result.kind,
     ): void {
         if (ruleResults) {
             for (const ruleResult of ruleResults) {
-                const partialFingerprints = this.getPartialFingerprintsFromRule(
-                    ruleResult,
-                );
-                // resultArray.push({
-                // ruleId: ruleResult.id,
-                // level: level,
-                // properties: {
-                // ...properties,
-                // tags: ['Accessibility'],
-                // },
-                // partialFingerprints: partialFingerprints,
-                // });
+                resultArray.push({
+                    ruleId: ruleResult.id,
+                    ruleIndex: ruleIdsToRuleIndices[ruleResult.id],
+                    kind: kind,
+                    level: this.getResultLevelFromResultKind(kind),
+                    message: {
+                        text: '',
```

Comment:
The sarif spec demands that the message be present and describe the result, so I think we'll probably have to come up with something more than an empty string here. Maybe `ruleResult.description` would be reasonable to start with?